src/app/store/recipes/loadRecipeEffect.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
```

Comment:
1.  use same file naming convention in project:`load-recipe-effect.ts`
2.  you can also add comma before `effect` like you have in other files in this folder: `load-recipe.effect.ts`
3. You can store multiple effects in this file so to keep it consistent with other files in folder so in summary it should be named `recipies.effects.ts`